modules/react/cookie-banner/README.md

```diff
@@ -15,28 +20,28 @@ Fixes a cookie banner to the bottom of the web page.
 Can be configured with a "Cookie Settings" element and a custom notice.
 
 ```tsx
-import CookieBanner from '@workday/canvas-kit-react/cookie-banner'
+import DeprecatedCookieBanner from '@workday/canvas-kit-react/cookie-banner'
```

Comment:
General question about deprecations: why do we change the import name? Why not just the console warning? The Deprecated Button is that way because we had a deprecated button and a non-deprecated button. We don't have a regular `CookieBanner` to designate it from the deprecated one.

Reply:
We're using it as a consistent identifier prefix for consumers, similar to React's `UNSAFE_componentWillReceiveProps`. They might upgrade to v6 now and need to migrate later. `Deprecated` helps teams grep for it.